bittensor/subtensor/client/__init__.py

```diff
@@ -195,10 +199,12 @@ async def get_stake_for_uid(self, uid) -> Balance:
             params = [uid]
         )
 
-        if not stake:
+        result = stake['result']
```

Comment:
would this fail if we got `stake = None`? Might be safer to say stake.get('result') that way if all of stake is None it won't bork?